plover/oslayer/winkeyboardcontrol.py

```diff
@@ -117,203 +118,150 @@ class WindowsKeyboardLayout:
-        'oslash': 248, 'acute': 237, 'ssharp': 223, 'Igrave': 204,
+        'oslash': 248, 'acute': 237, 'ssharp': 223, 'igrave': 204,
         'twosuperior': 178, 'udiaeresis': 252, 'notsign': 172, 'exclamdown': 161,
-        'ordfeminine': 9792, 'Otilde': 213, 'agrave': 224, 'ection': 167,
-        'egrave': 232, 'macron': 175, 'Icircumflex': 206, 'diaeresis': 168,
-        'ucircumflex': 251, 'atilde': 227, 'Acircumflex': 194, 'degree': 176,
-        'THORN': 222, 'acircumflex': 226, 'Aring': 197, 'Ooblique': 216,
-        'Ugrave': 217, 'Agrave': 192, 'ydiaeresis': 255, 'threesuperior': 179,
-        'Egrave': 200, 'Idiaeresis': 207, 'igrave': 236, 'ETH': 208,
-        'Ecircumflex': 202, 'Aacute': 193, 'cent': 162, 'registered': 174,
-        'Oacute': 211, 'Adiaeresis': 228, 'guillemotleft': 171, 'ediaeresis': 235,
-        'Ograve': 210, 'mu': 956, 'paragraph': 182, 'Ccedilla': 199, 'thorn': 254,
+        'ordfeminine': 9792, 'otilde': 213, 'agrave': 224, 'ection': 167,
+        'egrave': 232, 'macron': 175, 'icircumflex': 206, 'diaeresis': 168,
+        'ucircumflex': 251, 'atilde': 227, 'acircumflex': 194, 'degree': 176,
+        'thorn': 222, 'acircumflex': 226, 'aring': 197, 'ooblique': 216,
+        'ugrave': 217, 'agrave': 192, 'ydiaeresis': 255, 'threesuperior': 179,
+        'egrave': 200, 'idiaeresis': 207, 'igrave': 236, 'eth': 208,
+        'ecircumflex': 202, 'aacute': 193, 'cent': 162, 'registered': 174,
+        'oacute': 211, 'adiaeresis': 228, 'guillemotleft': 171, 'ediaeresis': 235,
+        'ograve': 210, 'mu': 956, 'paragraph': 182, 'ccedilla': 199, 'thorn': 254,
         'threequarters': 190, 'ae': 230, 'brokenbar': 166, 'nobreakspace': 32,
-        'currency': 164, 'ugrave': 249, 'Ucircumflex': 219, 'odiaeresis': 246,
-        'periodcentered': 183, 'Uacute': 218, 'idiaeresis': 239, 'yacute': 253,
-        'sterling': 163, 'AE': 198, 'Ediaeresis': 203, 'onequarter': 188,
-        'onehalf': 189, 'Thorn': 222, 'aacute': 225, 'icircumflex': 238,
-        'Udiaeresis': 220, 'eacute': 233, 'Eth': 240, 'eth': 240, 'Iacute': 205,
-        'onesuperior': 185, 'Ocircumflex': 212, 'Odiaeresis': 214, 'oacute': 243
+        'currency': 164, 'ugrave': 249, 'ucircumflex': 219, 'odiaeresis': 246,
+        'periodcentered': 183, 'uacute': 218, 'idiaeresis': 239, 'yacute': 253,
+        'sterling': 163, 'ae': 198, 'ediaeresis': 203, 'onequarter': 188,
+        'onehalf': 189, 'thorn': 222, 'aacute': 225, 'icircumflex': 238,
+        'udiaeresis': 220, 'eacute': 233, 'eth': 240, 'eth': 240, 'iacute': 205,
+        'onesuperior': 185, 'ocircumflex': 212, 'odiaeresis': 214, 'oacute': 243
     }
 
     # Maps from literal characters to their key names.
-    literals = collections.defaultdict(str, {
+    LITERALS = {
         '~': 'asciitilde', '!': 'exclam', '@': 'at',
-        '#': 'numbersign', '$': 'dollar', '%': 'percent',
+        '#': 'numbersign', '$': 'dollar', '%': 'percent', '^': 'asciicircum',
```

Comment:
I had an issue when writing the code originally where "^" send the Alt-Gr version instead of the Shift-6 version on a certain key layout, making stand alone "^"s impossible. Can you confirm that you can use the symbol without issue (curious about QWERTY, AZERTY, and the British layout)